src/main/groovy/org/mockito/release/notes/vcs/DefaultCommitApprover.java

```diff
@@ -0,0 +1,31 @@
+package org.mockito.release.notes.vcs;
+
+import org.mockito.release.notes.model.Commit;
+import org.mockito.release.notes.util.Predicate;
+
+import java.util.Collection;
+import java.util.LinkedList;
+
+public class DefaultCommitApprover implements Predicate<Commit> {
+
```

Comment:
Maybe CommitIgnored would be a better name since we use Ignore* predicates inside, and reversing logic makes it more complicated than it needs to be. Then it would be used like commitIgnored.isTrue(commit). Seems more readable.